src/main/java/gov/usgs/wma/waterdata/timeseries/TimeSeriesDao.java

```diff
@@ -8,13 +11,25 @@
 @Repository
 public class TimeSeriesDao extends SqlSessionDaoSupport {
 
+	private static final String COLLECTION_KEY = "collectionId"; // network ID
+	private static final String FEATURE_KEY    = "featureId";    // location ID
+	private static final String TIMESERIES_KEY = "timeseriesId"; // GUID
+
 	@Autowired
 	public TimeSeriesDao(SqlSessionFactory sqlSessionFactory) {
 		setSqlSessionFactory(sqlSessionFactory);
 	}
 
-	public String getTimeSeries(String monitoringLocationId, String timeSeriesId) {
+	public String getTimeSeries(String timeSeriesId) {
 		return getSqlSession().selectOne("groundwaterDailyValue.getGeoJson", timeSeriesId);
 	}
 
+	public String getTimeSeries(String collectionId, String featureId, String timeSeriesId) {
+		Map<String,String> params = new HashMap<>();
+		params.put(COLLECTION_KEY, collectionId);
+		params.put(FEATURE_KEY, featureId);
+		params.put(TIMESERIES_KEY, timeSeriesId);
+		return getSqlSession().selectOne("timeseries.getGeoJson", params);
```

Comment:
I didn't notice before that a new file/namespace was created. Technically, the query only works for groundwater (statistical) daily value time series. Also, now we have two versions of essentially the same query to maintain.
It would also be good to have integrations tests at this level (dao).

Reply:
Recall, I was asked not to "touch" the original. I figured that that mandate would be lifted at some point and then first file would be dropped. In this way, we do not have to look through and remove if-statements when we decide to remove that endpoint.

ok. will add direct tests. I didn't because coverage was satisfied by my controller tests.

Reply:
@duselmann I'm sorry if you got the impression that you shouldn't change the existing endpoints.  You are welcome to change/delete them, nothing currently depends on those endpoints.

Reply:
@dsteinich Should the timeseries dao (in the furture) have several different file/namespaces for each data type we have for timeseries? Should I change the original groundwater file?

@jkreft-usgs Yes, Jim. I was under the impression they were used in production and could not be touched. I was being careful to make sure they remained exactly as they were. I will work with DS to clean that up.

Reply:
I resolved this with the original namespace and a mybatis if statement.